test/notebooks/nbtest_sagemaker.py

```diff
@@ -5,13 +5,11 @@
 from inspect import cleandoc
 import textwrap
 
-import pytest
 from exasol.nb_connector.secret_store import Secrets
-from exasol.nb_connector.ai_lab_config import AILabConfig as CKey, StorageBackend
+from exasol.nb_connector.ai_lab_config import AILabConfig as CKey
 
 from notebook_test_utils import (
-    access_to_temp_secret_store,
-    access_to_temp_saas_secret_store,
+    backend_setup,
```

Comment:
We need to remove the fixture imports at some point, from TE we know this can provoke odd behavior

Reply:
The odd behaviour is caused by importing them twice - once by the pytest and the second time by us explicitly. Here pytest doesn't import fixtures declared in the `notebook_test_utils.py` so we are fine.

Reply:
ah ok, I think, we might need to document that a bit